### ISSUE TITLE
Error in rule create_cov_bed module pyd4 not found

### ISSUE THREAD

**Issue by tito-abbo** [harvardinformatics/snpArcher#233]
Hi snparcher team,
This appears to be same issue as closed report #144. Looking through that thread, I attempted the suggested workaround by trying to install pyd4 in the conda environment but the package fails to install. I was able to download pyd4 directly from github and install it manually. It works ok when I just try import it into python directly. But when I try to rerun snparcher with the --use-conda flag, the error switches from no module named pyd4 to: "li
ne 4, in <module>                                                                                                             
    from pyd4 import D4File,D4Builder                                                                                         
  File "/rhome/tabbo002/.conda/envs/snparcher/lib/python3.11/site-packages/pyd4/__init__.py", line 5, in <module>
    from .pyd4 import D4File as D4FileImpl, D4Iter, D4Builder as D4BuilderImpl, D4Writer as D4WriterImpl, D4Merger as D4Merger
Impl
ModuleNotFoundError: No module named 'pyd4.pyd4'
"
In addition to troubleshooting the bug, can you offer a suggestion to fix my workaround. It has taken over a month to run the pipeline up to this stage, so I am worried about possibly losing my progress.
Best,
Tito

**Comment by cademirch** [comment]
Yeah pyd4 is been a pain, sorry about that. I have a replacement, I'll try to get that merged soon.

**Comment by tito-abbo** [comment]
Oh I didn't even consider doing that. What are the possible adverse interpretation consequences of working with a dataset that hasn't been filtered in this way?

**Comment by cademirch** [comment]
Resolved by #244
